tests/test_briceletbaklava.py

```diff
@@ -0,0 +1,36 @@
+# mypy: allow-untyped-defs
+
+from recipe_scrapers.briceletbaklava import BricelEtBaklava
+from tests import ScraperTest
+
+
+class TestBricelEtBaklavaScraper(ScraperTest):
+
+    scraper_class = BricelEtBaklava
+
+    def test_host(self):
+        self.assertEqual("briceletbaklava.ch", self.harvester_class.host())
+
+    def test_author(self):
+        self.assertEqual('Michel/Bricelet & Baklava', self.harvester_class.author())
+
+    def test_title(self):
+        self.assertEqual('Bürli de Saint-Gall (St. Galler Bürli)', self.harvester_class.title())
+
+    def test_category(self):
+        self.assertEqual('Boulangerie\nRecettes de pains', self.harvester_class.category())
```

Comment:
Comma-separated categories _might_ be an improvement here -- or at least more consistent with other existing tests.

(it's slightly similar to the `ingredients` / `instructions` difference: this field could either be considered as a single string, or as a list of strings)